Cargo.toml

```diff
@@ -95,7 +95,7 @@ version = "=0.24.2"
 features = ["blob", "serde_json", "i128_blob", "bundled", "trace"]
 
 [dependencies.ed25519-dalek]
-version = "=1.0.0-pre.3"
+version = "=1.0.1"
 features = ["serde"]
 
 [dependencies.curve25519-dalek]
```

Comment:
Out of curiosity, why are we using Dalek?  My wrapped secp256k1 is significantly faster, and also maintains compatibility with bitcoin signatures.

Reply:
We use it to implement a VRF described here: https://datatracker.ietf.org/doc/id/draft-irtf-cfrg-vrf

Reply:
Do you want me to try replacing dalek with my secp wrappers?  It's basically API compatible, by intent.

Reply:
I don't think it's high priority.  There's only one VRF proof generated per miner per block produced, and there is only one VRF proof verification per Stacks block, so a performance speed-up won't do very much for overall block-processing performance.  Also, any new implementation needs to be "bug-for-bug" compatible with the current implementation, since VRF proof creation and verification are on consensus-critical code paths.